packages/quantic/cypress/integration/facets/category-facet/catgory-facet.cypress.ts

```diff
@@ -0,0 +1,200 @@
+  sortCriteria: string;
+  withSearch: boolean;
+  isCollapsed: boolean;
+}
+
+describe('quantic-category-facet', () => {
+  const pageUrl = 's/quantic-category-facet';
+
+  const defaultField = 'geographicalhierarchy';
+  const defaultLabel = 'Country';
+  const defaultNumberOfValues = 8;
+
+  function visitCategoryFacetPage(
+    options: Partial<CategoryFacetOptions> = {},
+    waitForSearch = true
+  ) {
+    interceptSearch();
+    cy.visit(pageUrl);
+    configure(options);
+    if (waitForSearch) {
+      cy.wait(InterceptAliases.Search);
+    }
+  }
+  function setupWithDefaultSettings() {
+    visitCategoryFacetPage(
+      {
+        field: defaultField,
+        label: defaultLabel,
+        numberOfValues: defaultNumberOfValues,
+      },
+      false
+    );
+  }
+
+  function setupGoDeeperOneLevel() {
+    setupWithDefaultSettings();
+    Actions.selectChildValueAt(canadaHierarchyIndex[0]);
+  }
+
+  function setupGoDeeperTwoLevels() {
+    setupWithDefaultSettings();
+    Actions.selectChildValueAt(canadaHierarchyIndex[0]);
+    interceptSearch();
+    Actions.selectChildValueAt(canadaHierarchyIndex[1]);
+    interceptSearch();
+  }
+
+  function setupShowMore() {
+    setupWithDefaultSettings();
+
+    setupGoDeeperOneLevel();
+    Actions.clickShowMoreButton();
+  }
+
+  function setupShowLess() {
+    setupShowMore();
+    interceptSearch();
+    Actions.clickShowLessButton();
+  }
+
+  function setupGoDeeperFourLevels() {
+    setupWithDefaultSettings();
+    Actions.selectChildValueAt(canadaHierarchyIndex[0]);
+    interceptSearch();
+    Actions.selectChildValueAt(canadaHierarchyIndex[1]);
+    interceptSearch();
+    Actions.selectChildValueAt(canadaHierarchyIndex[2]);
+    interceptSearch();
+    Actions.selectChildValueAt(canadaHierarchyIndex[3]);
+    interceptSearch();
+  }
+
+  describe('with default category facet', () => {
+    it('should work as expected', () => {
+      setupWithDefaultSettings();
+
+      Expect.isAccesssible(true);
+      Expect.displayLabel(true);
+      Expect.displayFacetCount(true);
+      Expect.displaySearchInput(false);
+      Expect.numberOfValues(7);
+    });
+
+    describe('when selecting on the 1st level data set', () => {
+      it('should bold the selected parent level', () => {
+        const selectedPath = canadaHierarchy.slice(0, 1);
+        setupGoDeeperOneLevel();
+
+        Expect.numberOfParentValues(1);
+        Expect.parentValueLabel(selectedPath[0]);
+        Expect.numberOfChildValues(8);
+        Expect.pathInUrl(selectedPath);
+        Expect.logCategoryFacetSelected(selectedPath);
+      });
+    });
+
+    describe('when selecting value subsequently to go next deeper level', () => {
+      it('should bold next child level', () => {
+        const selectedPath = canadaHierarchy.slice(0, 2);
+        setupGoDeeperTwoLevels();
+
+        Expect.numberOfParentValues(1);
+        Expect.pathInUrl(selectedPath);
+        // Expect.logCategoryFacetSelected(selectedPath);
```

Comment:
Is it the broken expectation we talked about earlier?

Reply:
yes it is, I m trying to change the wait , still with no success, but I know I need to wait for it 😆